tests/by-util/test_cp.rs

```diff
@@ -456,6 +456,30 @@ fn test_cp_arg_interactive_update() {
         .no_stdout();
 }
 
+#[test]
+#[cfg(not(target_os = "android"))]
+fn test_cp_arg_interactive_verbose() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("a");
+    at.touch("b");
+    ucmd.args(&["-vi", "a", "b"])
+        .pipe_in("N\n")
+        .fails()
+        .stdout_is("skipped 'b'\n");
+}
+
+#[test]
+#[cfg(not(target_os = "android"))]
+fn test_cp_arg_interactive_verbose_clobber() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("a");
+    at.touch("b");
+    ucmd.args(&["-vin", "a", "b"])
+        .pipe_in("y\n")
```

Comment:
As there is no prompt in this test case, it's not relevant what you pipe in, so `pipe_in` is not needed.
```suggestion
```